src/Components/Calculator.js

```diff
@@ -1,36 +1,62 @@
+    }
+    this.setState({
+      total,
+      next,
+      operation,
+      display,
+    });
   }
 
   render() {
+    const { display } = this.state;
     return (
       <section className="calculator-container">
-        <p className="display">0</p>
-        <NumbButton addClass="AC" element="AC" />
-        <NumbButton addClass="mm" element="+/-" />
-        <NumbButton addClass="porcentaje" element="%" />
-        <NumbButton addClass="nueve" element="9" />
-        <NumbButton addClass="ocho" element="8" />
-        <NumbButton addClass="siete" element="7" />
-        <NumbButton addClass="seis" element="6" />
-        <NumbButton addClass="cinco" element="5" />
-        <NumbButton addClass="cuatro" element="4" />
-        <NumbButton addClass="tres" element="3" />
-        <NumbButton addClass="dos" element="2" />
-        <NumbButton addClass="uno" element="1" />
-        <NumbButton addClass="cero" element="0" />
-        <NumbButton addClass="decimal" element="." />
-        <NumbButton addClass="division" element="/" />
-        <OperatorButton addClass="multiplicar" element="x" />
-        <OperatorButton addClass="restar" element="-" />
-        <OperatorButton addClass="sumar" element="+" />
-        <OperatorButton addClass="igual" element="=" />
+        <p className="display">{display}</p>
+        <NumbButton addClass="AC" element="AC" functionClick={this.eventClick} />
+        <NumbButton addClass="mm" element="+/-" functionClick={this.eventClick} />
+        <NumbButton addClass="porcentaje" element="%" functionClick={this.eventClick} />
+        <NumbButton addClass="nueve" element="9" functionClick={this.eventClick} />
+        <NumbButton addClass="ocho" element="8" functionClick={this.eventClick} />
+        <NumbButton addClass="siete" element="7" functionClick={this.eventClick} />
+        <NumbButton addClass="seis" element="6" functionClick={this.eventClick} />
+        <NumbButton addClass="cinco" element="5" functionClick={this.eventClick} />
+        <NumbButton addClass="cuatro" element="4" functionClick={this.eventClick} />
+        <NumbButton addClass="tres" element="3" functionClick={this.eventClick} />
+        <NumbButton addClass="dos" element="2" functionClick={this.eventClick} />
+        <NumbButton addClass="uno" element="1" functionClick={this.eventClick} />
+        <NumbButton addClass="cero" element="0" functionClick={this.eventClick} />
+        <NumbButton addClass="decimal" element="." functionClick={this.eventClick} />
+        <NumbButton addClass="division" element="/" functionClick={this.eventClick} />
+        <OperatorButton addClass="multiplicar" element="x" functionClick={this.eventClick} />
+        <OperatorButton addClass="restar" element="-" functionClick={this.eventClick} />
+        <OperatorButton addClass="sumar" element="+" functionClick={this.eventClick} />
+        <OperatorButton addClass="igual" element="=" functionClick={this.eventClick} />
       </section>
```

Comment:
[OPTIONAL] Instead of all these lines of code, I would advise you to add the element in an array and then map it out with the javascript map keyword. It would not take you more than 5 lines of code instead of this long line of code. In fact, it is the professional and industrial standard. Try it out.